fast64_internal/f3d/f3d_enums.py

```diff
@@ -254,22 +254,22 @@
 ]
 
 enumBlendColor = [
-	('G_BL_CLR_IN', 'Input', 'Input'),
-	('G_BL_CLR_MEM', 'Memory', 'Memory'),
-	('G_BL_CLR_BL', 'Blend', 'Blend'),
-	('G_BL_CLR_FOG', 'Fog', 'Fog')
+	('G_BL_CLR_IN', 'Input (CC/Blender)', 'First cycle: Color Combiner RGB, Second cycle: Blender numerator from first cycle'),
+	('G_BL_CLR_MEM', 'Framebuffer', 'Framebuffer (Memory)'),
+	('G_BL_CLR_BL', 'Blend', 'Blend Register'),
+	('G_BL_CLR_FOG', 'Fog', 'Fog Register')
```

Comment:
```suggestion
	('G_BL_CLR_BL', 'Blend Color', 'Blend Color Register'),
	('G_BL_CLR_FOG', 'Fog Color', 'Fog Color Register')
```
the manual likes to call thse registers "color registers" (eg in http://n64devkit.square7.ch/n64man/gdp/gDPSetFogColor.htm )
and I think it's clearer anyway to mention "Color"